README.md

```diff
@@ -29,48 +29,12 @@ This SDK supports:
 * UIKit
 * SwiftUI
 
-## Access Token
+## Client ID
 
-The PayPal SDK uses access tokens for authentication.
+The PayPal SDK uses client ID for authentication.
 
 > The following example can be adapted to any server-side language/framework of your choice. We use command-line curl requests to demonstrate the overall composition of the Access Token HTTP request.
```

Comment:
```suggestion
```

Reply:
Addressed in 44250f07b9791b6cac9cca79803900a17b1d10ee